### PR TITLE
Converted documentationURL & evidenceStandardSupport into FieldArrays 

### PR DESCRIPTION
**Pull request by nathanbaleeta**
Converted `documentationURL` & `evidenceStandardSupport` into arrays using `fieldArray` component

### CODE REVIEW

File: schemas/schema.js [hunk 1]
```diff
@@ -1064,11 +1065,13 @@ const schema = {
                 {
                   type: "required",
                 },
+              ],
+              fields: [
                 {
-                  type: validatorTypes.URL,
```
Comment (lacabra) [inline schemas/schema.js:1068]: @nathanbaleeta can you confirm that you remove the URL validator because you expect text input in addition to URLs?

Comment (lacabra) [reply schemas/schema.js:1068]: If so, we may need to update the text to reflect the fact that we are expecting both URL and text.

File: schemas/schema.js [hunk 2]
```diff
@@ -1064,11 +1065,13 @@ const schema = {
...
+                  component: "text-field",
+                  label: "Documentation URL",
                 },
               ],
-              classes: {root: "conditional"},
```
Comment (lacabra) [inline schemas/schema.js:1071]: Why is the conditional class removed?

Comment (nathanbaleeta) [reply schemas/schema.js:1071]: Fixed in commit bca3f52.

File: schemas/schema.js (hunk 1 above)
Comment (nathanbaleeta) [reply schemas/schema.js:1068]: The API threw an error on adding the above validation hence reason for dropping it. I will update the text to indicate the field captures both URL and text.

Comment (nathanbaleeta) [reply schemas/schema.js:1068]: The API throws an error on adding the above validation for URL when using the `fieldArray` component sometimes. For now will resort to indicating on the label that this field accepts both text and URLs.

Comment (lacabra) [reply schemas/schema.js:1068]: @nathanbaleeta Can you link to the error, please?

Comment (nathanbaleeta) [reply schemas/schema.js:1068]: Resolved on #21

Comment (lacabra) [reply schemas/schema.js:1068]: Resolved in 6a6f8f2

**Review by lacabra — APPROVED** [review]
This looks good to me.

**Review by lacabra — APPROVED** [review]

**Comment by nathanbaleeta** [comment]
I have added URL validation to `standards[evidenceStandardSupport]` field. Changes feature on this commit b58b2f3. @lacabra ready for your review.

**Pull request merged**